lib/tic_tac_toe/web/game_presenter.ex

```diff
@@ -0,0 +1,11 @@
+defmodule TicTacToe.Web.GamePresenter do
+
+  def render_game(game) do
+    board_width = Board.width(game.board)
+    rows = game.board.cells |> Enum.with_index |> Enum.chunk(board_width)
```

Comment:
Indeed, that's what I was asking as I did this, but then failed to actually go through with it. There already is a private get_rows method in board, so I just need to expose it and add tests to cover it :)